windows-setup.md

```diff
@@ -1,34 +1,7 @@
-# Windows Powershell Setup Instructions
+# Windows Prerequisites
 
-## Prerequisites
+1. [Powershell](https://docs.microsoft.com/en-us/powershell/)
+2. Download Python version >= 2.7.13 from [python.org](https://www.python.org/downloads/windows/).
+3. Install both Python and pip to PATH either in installer (enable component during the installation) or manually. The easiest way to do this is to make sure the below is checked during installation: ![readme_add_python_to_path_installer](readme_add_python_to_path_installer.png)

```

Comment:
This screenshot is the 2.7 installer

Here's what it looks like in 3: https://docs.python.org/3/_images/win_installer.png

So it's a checkbox to click instead of a dropdown. Still not checked by default though for some reason